helpers/make_network_dirs_main.py

```diff
@@ -0,0 +1,171 @@
+"""Script that creates all directories"""
+import sys
+sys.path.append("D:/research-and-development")
+from src.utils.helpers import tree_to_list
+from src.utils.local_file_mods import rd_file_exists
+import os
+
+# Change to the project repository location
+my_wd = os.getcwd()
+my_repo = "R:\\"
+if not my_wd.endswith(my_repo):
+    os.chdir(my_repo)
+
+
+def make_dir(dir):
+    """Creates a directory bassed on the provided folder_path."""
+    # create directory if it does not exist
+
+    if rd_file_exists(dir, raise_error=False) is False:
+        os.makedirs(dir)
+        print(f"Created {dir}")
+    else:
+        print(f"Directory {dir} already exists.")
+
+
+def run_make_dirs():
+    """Creates all directories based on the `tree` structure.
+
+    tree (dict) converted to list of lists by function src.utils.helpers.tree_to_list.
+    make_dir function is called for item in the list of directories.
+
+    """
+
+    root = "BERD Results System Development 2023/DAP_emulation"
+
+    tree = {
+        "2022_surveys": {
+            "PNP": {
```

Comment:
Is there a way to do this where you don't have to hard code survey (PNP / BERD) or year, and you can put these in at the top?

Reply:
I gave addressed this comment by adding a loop into the function that iterates through years and survey types.